paddle/fluid/operators/pool_mkldnn_op.cc

```diff
@@ -0,0 +1,219 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "mkldnn.hpp"
+
+#include "paddle/fluid/operators/pool_op.h"
+#include "paddle/fluid/platform/mkldnn_helper.h"
+
+namespace paddle {
+namespace operators {
+
+template <typename T>
+class PoolMkldnnKernel : public paddle::framework::OpKernel<T> {
```

Comment:
`PoolMKLDNNKernel` ==> `PoolMKLDNNOpKernel`

Name style should keep same as [cudnn](
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/pool_cudnn_op.cu.cc#L29)

What do you say @luotao1, is that necessary to solve in this PR?

same as `ConvOpMkldnnKernel`

Reply:
I think this can be solved in this PR, since there are only two Kernels which should be renamed now.

Reply:
done
I additionally added commit with refactoring of mkldnn convolution to align to the pooling kernel style.